hpedockerplugin/cmd/cmd_createfpg.py

```diff
@@ -25,13 +25,23 @@ def execute(self):
             LOG.info("Creating FPG %s on the backend using CPG %s" %
                      (self._fpg_name, self._cpg_name))
             try:
-                self._mediator.create_fpg(self._cpg_name, self._fpg_name)
+                config = self._file_mgr.get_config()
+                fpg_size = FPG_SIZE
+                if config.hpe3par_default_fpg_size:
+                    fpg_size = int(config.hpe3par_default_fpg_size)
```

Comment:
Here ValueError exception should be catched. If it is taken care during the initialization of a backend then it is okay.